link/src/main/java/com/stripe/android/link/ui/LinkButtonView.kt

```diff
@@ -43,8 +43,8 @@ import com.stripe.android.link.theme.linkColors
 
 private val LinkButtonVerticalPadding = 6.dp
 private val LinkButtonHorizontalPadding = 10.dp
-private val LinkButtonShape = RoundedCornerShape(22.dp)
-private val LinkButtonEmailShape = RoundedCornerShape(16.dp) // Button corner radius - padding
+private val LinkButtonShape = RoundedCornerShape(4.dp)
+private val LinkButtonEmailShape = RoundedCornerShape(4.dp) // Button corner radius - padding
```

Comment:
The comment is outdated now, and the math for the nested corner radius would also not hold anymore. @jarvis-stripe, what’s the right corner radius for this inner shape?